server/models.py

```diff
@@ -77,7 +77,7 @@ class StatesUTs(models.TextChoices):
         UK = "UK", _("Uttarakhand")
         WB = "WB", _("West Bengal")
 
-    state_ut = models.CharField(max_length=5, choices=StatesUTs.choices)
+    state_ut = models.CharField(max_length=5, choices=StatesUTs.choices, blank=True)
```

Comment:
I don't think `blank=True` is the best way to handle this. It will allow incorrect inputs, even for people in India. I think we can copy the "I'm not in India" that was in the original Google form.